pkg/plugin/aggregation/status.go

```diff
@@ -42,7 +46,7 @@ type Status struct {
 }
 
 func (s *Status) updateStatus() error {
-	status := CompleteStatus
+	status := PostProcessingStatus
```

Comment:
The aggregator itself now doesn't ever set the CompleteStatus, since it is just responsible for aggregating the data from the plugins. As a whole, sonobuoy has to signoff that everything is ready for download before we report that.